src/gc/gc.cpp

```diff
@@ -26879,26 +26818,10 @@ uint32_t gc_heap::bgc_thread_function()
     BOOL do_exit = FALSE;
     Thread* thread_to_destroy = 0;
```

Comment:
It would be nice to move the DestroyThread call to the wrapper function as well, and remove DestroyThread from the gcenv headers (the other call to DestroyThread in `gc_heap::kill_gc_thread` is completely broken - it is not causing problems because this function is never called).